docs/tutorials/jokes.md

```diff
@@ -1289,22 +1289,17 @@ export default function JokesRoute() {
 
 <summary>app/routes/index.tsx</summary>
 
-```tsx filename=app/routes/index.tsx lines=[1,4,6-13]
+```tsx filename=app/routes/index.tsx lines=[2,16-31]
```

Comment:
```suggestion
```tsx filename=app/routes/index.tsx lines=[2,11-26]
```

Reply:
Just spotted this too! Fixed now